app-misc/double-commander/double-commander-1.0.9.ebuild

```
@@ -0,0 +1,68 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit xdg-utils
+
+DESCRIPTION="Cross Platform file manager."
```

https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide/Style_Guide#Descriptive_text

```suggestion
DESCRIPTION="Cross Platform file manager"
```